Assets/ScriptableObjects/AttackData.cs

```diff
@@ -21,3 +22,4 @@ public enum AttackType {Melee, Projectile};
     public bool knockback; 
```

Comment:
Defining the amount of knockback force here in `AttackData` might be useful as well, or it might be more simple to define a single common knockback force that gets used in attacks.